app/gw2/client.py

```
@@ -88,29 +96,26 @@ async def _get(self, endpoint: str, params: dict | None = None, require_token: b
             except httpx.RequestError as e:
                 retries += 1
                 if retries > self.max_retries:
-                    raise RuntimeError(f"Conection error after {self.max_retries} attemps: {e}")
+                    raise RuntimeError(f"Connection error after {self.max_retries} attempts.") from e
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)